.github/workflows/ci.yml

```diff
@@ -42,17 +42,16 @@ jobs:
       matrix:
         include:
           # Linux, gcc
-          - { compiler: gcc-5, cxxstd: '14',       boostBranch: boost-1.65.0, os: ubuntu-22.04, container: 'ubuntu:18.04' }
-          - { compiler: gcc-5, cxxstd: '14',       boostBranch: boost-1.77.0, os: ubuntu-22.04, container: 'ubuntu:18.04' }
-          - { compiler: gcc-12,cxxstd: '14,17,20', boostBranch: master,       os: ubuntu-22.04 }
-          
+          - { compiler: gcc-12, cxxstd: '14,17,20', boostBranch: boost-1.86.0, os: ubuntu-24.04 }
+          - { compiler: gcc-14, cxxstd: '14,17,20', boostBranch: boost-1.86.0, os: ubuntu-24.04 }
+          - { compiler: gcc-14, cxxstd: '14,17,20', boostBranch: master, os: ubuntu-24.04 }
+
           # Linux, clang
-          - { compiler: clang-5.0, cxxstd: '14',       boostBranch: boost-1.65.0, os: ubuntu-22.04, container: 'ubuntu:18.04' }
-          - { compiler: clang-5.0, cxxstd: '14',       boostBranch: boost-1.77.0, os: ubuntu-22.04, container: 'ubuntu:18.04' }
-          - { compiler: clang-14,  cxxstd: '14,17,20', boostBranch: master,       os: ubuntu-22.04 }
+          - { compiler: clang-14,  cxxstd: '14,17,20', boostBranch: boost-1.86.0, os: ubuntu-24.04 }
+          - { compiler: clang-18,  cxxstd: '14,17,20', boostBranch: boost-1.86.0, os: ubuntu-24.04 }
+          - { compiler: clang-18,  cxxstd: '14,17,20', boostBranch: master, os: ubuntu-24.04 }
 
-          - { name: Collect coverage, coverage: yes,
-              compiler: gcc-8,     cxxstd: '14',       boostBranch: master,       os: ubuntu-20.04 }
+          - { name: Collect coverage, coverage: yes, compiler: gcc-14, cxxstd: '14', boostBranch: boost-1.86.0, os: ubuntu-24.04 }
```

Comment:
I remember having issues using other versions of GCC/gcov hence I stuck to version 8 here and in other projects